internal/gateway/frontend/testutil.go

```diff
@@ -83,8 +83,8 @@ func (b *mockBackend) CancelLease(ctx context.Context, tokenStr string) error {
 	return nil
 }
 
-func (b *mockBackend) CommitLease(ctx context.Context, tokenStr, oldRootHash, newRootHash string, tag gw.RepositoryTag) error {
-	return nil
+func (b *mockBackend) CommitLease(ctx context.Context, tokenStr, oldRootHash, newRootHash string, tag gw.RepositoryTag) (uint64, error) {
+	return 0, nil
```

Comment:
In the mock_receiver we return 1, here we return 0.

Maybe it is better to be consistent. I would support returning 1 also here.